resources/sts/4.11/managed/ROSACloudCredentialOperator.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "sts:GetCallerIdentity"
```

Comment:
Comparing to https://github.com/openshift/managed-cluster-config/blob/master/resources/sts/4.12/openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy.json, is this sufficient?